### ISSUE TITLE
Get network interface statistics

### ISSUE THREAD

**Issue by ManUnix** [elastic/gosigar#27]
Hi,

I'd like to collect network interface statistics with topbeat (someone has already opened an enhancement request: https://github.com/elastic/beats/issues/640).

In order to implement the network statistics in topbeat it might be helpful to get the absolute values (bytes/packets sent/received, errors, etc.) via gosigar.

Is the code already in the making? If not I'd be happy to contribute and create a PR for the network interface stats.

**Comment by andrewkroh** [comment]
The code is not already in the making, but I am considering using [gopsutil](https://github.com/shirou/gopsutil/tree/master/net) for the task. Take a look at the data produced by that library let me know if you think there is anything missing.

I already did some [experiments](https://github.com/andrewkroh/beats/blob/feature/system-modules/metricbeat/module/system/net/io.go) with using gopsutil to fetch network interface statistics in Metricbeat, which is probably where this network IO statistics feature will land. All the other topbeat metrics have already been added to Metricbeat.
